spec/data-types-and-values.html

```diff
@@ -170,7 +186,8 @@ <h1>
         </h1>
         <dl class="header"></dl>
         <emu-alg>
-          1. Let _valueString_ be ? ToString(_argument_.[[Value]]).
+          1. Let _value_ be ? Unbox(_argument_).

```

Comment:
This may cause issues with virtualization of `Box.unbox` in compartments, I need to think more about it.

Reply:
I can make it return just `Box()`.

Reply:
Not with the virtualization a Compartment would do to Box. See https://github.com/tc39/proposal-record-tuple/issues/254#issuecomment-932325997

```js
function toString() {
  const target = unwrap(this);
  // Perform https://tc39.es/ecma262/#sec-ordinarytoprimitive
  for (const methodName of ['toString', 'valueOf']) {
    const method = target[methodName];
    if (typeof method === 'function') {
      const result = method.call(target);
      if (Object(result) !== result) {
        return result;
      }
    }
  }
  throw new TypeError();
}
```

Looking at this I probably forgot the fallback to `toStringTag`, but that's a small detail.

Reply:
Ignore above comment, I the virtualization of `toString` is not possible.